modules/admin-credentials-root-secret-formats.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/manually-creating-iam.adoc
+
+ifeval::["{context}" == "manually-creating-iam-aws"]
+:aws:
+endif::[]
+ifeval::["{context}" == "manually-creating-iam-azure"]
+:azure:
+endif::[]
+ifeval::["{context}" == "manually-creating-iam-gcp"]
+:google-cloud-platform:
+endif::[]
+
+[id="admin-credentials-root-secret-formats_{context}"]
+= Admin credentials root secret format
+
+Each cloud provider uses a credentials root secret in the `kube-system`
+namespace by convention, which is then used to satisfy all `CredentialsRequests`
+and create their respective secrets. This is done either by minting new
+credentials, _Mint Mode_, or by copying the credentials root secret,
+_Passthrough Mode_.
+
+The format for the secret varies by cloud, and is also used for each
+`CredentialsRequest` secret.
+
+ifdef::aws[]
+
+.Amazon Web Services (AWS) secret format
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: aws-creds
+data:
+  aws_access_key_id: Base64encodeAccessKeyID
```

Comment:
@dgoodwin Should we have  Base64encodeAccessKeyID and similar values in each of the cloud providers' secret formatted to be a replaceable, or does the YAML explicitly look like this? Trying to determine if we have the correct style here. Thanks!

Reply:
An alternative would be using stringData instead of data, in which case the base64 encoding is not necessary. 

As written seems fine assuming people know that Base64encodeAccessKeyID means to base64 encode their access key ID.

Reply:
Yeah, stringData should be fine. It's when you 'oc get' the secret that you'll see the base64 representation.